contracts/VestingEscrowSimple.vy

```diff
@@ -70,23 +62,24 @@ def initialize(
     @param end_time Time until everything should be vested
     @param cliff_length Duration after which the first portion vests
     """
-    assert not self.initialized  # dev: can only initialize once
-    self.initialized = True
-
     self.token = ERC20(token)
     self.admin = admin
     self.start_time = start_time
     self.end_time = end_time
     self.cliff_length = cliff_length
 
-    assert self.token.transferFrom(msg.sender, self, amount)  # dev: could not fund escrow
-
     self.recipient = recipient
     self.disabled_at = end_time  # Set to maximum time
     self.total_locked = amount
-    log Fund(recipient, amount)
 
-    return True
+
+@external
+def seed(amount: uint256):
+    # probably should guard this one
```

Comment:
?

Reply:
same as before, i'd rather store that a contract has been initialized and not allow repeating any actions that are only done once.